source/extensions/tracers/opentelemetry/samplers/dynatrace/dynatrace_sampler.cc

```diff
@@ -73,6 +72,26 @@ class DynatraceTag {
   uint32_t path_info_;
 };
 
+// add Dynatrace specific span attributes
+void addSamplingAttributes(uint32_t sampling_exponent,
+                           std::map<std::string, std::string>& attributes) {
+
+  const auto multiplicity = SamplingState::toMultiplicity(sampling_exponent);
+  // The denominator of the sampling ratio. If for example the Dynatrace OneAgent samples with a
+  // probability the value of supportability.atm_sampling_ratio would be 16
```

Comment:
```suggestion
  // probability the value of supportability.atm_sampling_ratio would be 16
```
Hum why would it be 16? Not sure if I follow it